sdk/cosmosdb/cosmos/test/public/functional/queryIterator.spec.ts

```diff
@@ -53,13 +53,15 @@ describe("Correlated Activity Id", function () {
 
   it("fetchNext and fetchAll should have different correlated id with same iterator", async () => {
     const queryIterator = container.items.readAll();
-    await queryIterator.fetchNext();
+    const fetchNextResult = await queryIterator.fetchNext();
```

Comment:
Could we also include a test case for error scenarios? This would involve testing scenarios where fetchNext or fetchAll fails for some reason, and we verify/assert the correlated activity ID.

Reply:
addressed